app/jobs/sync/channel_promo_registrations_stats_job.rb

```diff
@@ -1,10 +1,10 @@
 # Fetches and saves the referral stats for channel owned codes
 class Sync::ChannelPromoRegistrationsStatsJob < ApplicationJob
   include PromosHelper
-  queue_as :heavy
 
   def perform
-    promo_registrations = PromoRegistration.channels_only
-    Promo::RegistrationsStatsFetcher.new(promo_registrations: promo_registrations).perform
+    active_publisher_ids = Publisher.not_suspended
```

Comment:
I think it is okay to update these stats only for active publishers, but would like to get a second opinion. @nvonpentz what do you think?

Reply:
It should be fine to update only for active users